### ISSUE TITLE
Listeners for device parameters

### ISSUE THREAD

**Issue by MartinSahlen** [ideoforms/AbletonOSC#107]
Currently we can get and set device parameters but there are no listeners. Would like to have that!

**Comment by MartinSahlen** [comment]
Love the project though, will see if I can look at this myself.

**Comment by ben-juodvalkis** [comment]
agreed, though there is this: Changes for any Parameter property can be listened for by calling /live/device/start_listen/parameter/value <track_index> <device index> <parameter_index>

**Comment by MartinSahlen** [comment]
Not sure that was there at this point, seems it was added after https://github.com/ideoforms/AbletonOSC/pull/114? Either way I rolled my own.
